Makefile

```diff
@@ -85,6 +115,17 @@ protogen_show:
 protogen_clean:
 	find . -name "*.pb.go" | grep -v -e "prototype" -e "vendor" | xargs rm
 
+# TODO(team): Add more protogen targets here.
+.PHONY: protogen_local
+## V1 Integration - Use `protoc` to generate consensus .go files from .proto files.
+protogen_local:
+	$(eval proto_dir = "./shared/types/proto/")
+
+	protoc -I=${proto_dir} -I=./shared/types/proto --go_out=./shared ./shared/types/proto/*.proto
+	protoc -I=${proto_dir} -I=./consensus/types/proto --go_out=./consensus ./consensus/types/proto/*.proto
```

Comment:
So there's some consensus module specific logic that makes me a bit uncomfortable, any way we can make a global convention for .proto?

Reply:
Resolving because some more global changes/cleanup/refactoring outdate this comment.